src/processing/dom/html-namespaces.js

```diff
@@ -0,0 +1,13 @@
+// -------------------------------------------------------------
+// WARNING: this file is used by both the client and the server.
+// Do not use any browser or node-specific API!
+// -------------------------------------------------------------
+
+export default {
+    HTML:   'http://www.w3.org/1999/xhtml',
+    MATHML: 'http://www.w3.org/1998/Math/MathML',
+    SVG:    'http://www.w3.org/2000/svg',
+    XLINK:  'http://www.w3.org/1999/xlink',
+    XML:    'http://www.w3.org/XML/1998/namespace',
+    XMLNS:  'http://www.w3.org/2000/xmlns/'
+};
```

Comment:
#477 is related to XML

> For future use.

Never add dead code based on assumption. Let's add namespace when it they will be required.